### DISCUSSION TITLE
Start of a tutorial showing how Blawx interfaces with s(CASP)

### DISCUSSION THREAD

**Discussion by DavidPratten** [Lexpedite/blawx#324] [Ideas]
Hi Jason,

I've created a SWISH notebook that shows the flow from a law, through its encoding in Blawx, and translation to s(CASP).

[Law - Blawx - s(CASP)](https://swish.swi-prolog.org/p/rockpaperscissorslaw%205fe0.swinb)

Your feedback is welcome.  Feel free to fork the SWISH if you have ideas for improving it.

Would this be a candidate for a Tutorial page for this project?

David

**Comment by Gauntlet173** [comment]
Hi, David. Taking a quick look at the page, it seems like you have nailed the translation from law to Blawx to s(CASP). I'm not sure if it's within the scope of what I'm trying to do with Blawx to include this as a tutorial for Blawx. The point of Blawx is to remove the necessity for learning how to write the s(CASP) yourself. This would seem on its face to be more useful for people who hoped to contribute to Blawx, to help them understand what Blawx is doing behind the scenes, which is also extremely useful. But I don't really have a place for that to "live", yet. Suggestions are welcome.
